server/src/main/java/org/elasticsearch/common/time/DateFormatters.java

```diff
@@ -893,8 +893,7 @@
     /*
      * Returns a formatter for parsing the seconds since the epoch
      */
-    private static final DateFormatter EPOCH_SECOND = new JavaDateFormatter("epoch_second",
-        new DateTimeFormatterBuilder().appendValue(ChronoField.INSTANT_SECONDS).toFormatter(Locale.ROOT));
+    private static final DateFormatter EPOCH_SECOND = EpochSecondsDateFormatter.INSTANCE;
```

Comment:
Why do we need the constant now, we can just reference the singleton in the one place this is used?